### ISSUE TITLE
Tree column parent cell should include `aria-expanded` value so that row collapsed state is announced

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#4643]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=51&t=21319)

Hi,

We are using the Tree Grid and have noticed a couple of accessibility issues relating to the tree column.

Screen readers cannot read whether a tree row is expanded or collapsed.
The icons added to grid rows (using iconCls on GridRowModel) do not have any alternate text that's readable by screen readers. We need to provide an accessible alternative to icons.

Thanks.

Example here: https://www.w3.org/TR/wai-aria-practices/examples/treegrid/treegrid-1.html
